.storybook/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["@babel/preset-typescript", "@babel/preset-env", "@babel/preset-react"],
-  "plugins": ["react-require"]
+  "plugins": ["react-require", "babel-plugin-styled-components"]
```

Comment:
Wish I knew this sooner —  adding the babel plugin here lets us have the nice component names in the rendered classes in storybook too (previously this only worked on the dev server)